src/Sulu/Bundle/AdminBundle/Resources/js/components/ImageRectangleSelection/ImageRectangleSelection.js

```diff
@@ -126,6 +126,8 @@ class ImageRectangleSelection extends React.Component<Props> {
 
         return (
             <RectangleSelection
+                containerHeight={this.scaledImageHeight}
+                containerWidth={this.scaledImageWidth}
```

Comment:
I don't know

Reply:
I am not sure, but my guess would be that's because you've changed the import in this file, which I would not do, unless you have a specific reason for that.

Reply:
fixed